cibuildwheel/linux.py

```diff
@@ -42,11 +42,13 @@ def build(project_dir, output_dir, test_command, test_requires, before_build, bu
 
     python_configurations = get_python_configurations(build_selector)
     platforms = [
-        ('manylinux1_x86_64', manylinux1_images.get('x86_64') or 'quay.io/pypa/manylinux1_x86_64'),
-        ('manylinux1_i686', manylinux1_images.get('i686') or 'quay.io/pypa/manylinux1_i686'),
+        ('manylinux_x86_64', manylinux_images['x86_64']),
+        ('manylinux_i686', manylinux_images['i686']),
     ]
 
     for platform_tag, docker_image in platforms:
+        if not docker_image:
+            continue
```

Comment:
See above comment on the empty `manylinux_i686_image` setting. Builds should be skipped using CIBW_SKIP, not with empty image settings.